portfolio/src/main/webapp/script2.js

```diff
@@ -0,0 +1,12 @@
+/*
+*This talks about me!
+*/
+
+function addBackground()  {
+
+    const info = document.getElementById('info-container');
+    if (info.innerText == '')
```

Comment:
fyi: another way to do this toggling is to create a css class called something like "hidden" with an attribute `display: none`. Then you can check if it has the class and remove it or add it if it does not. Then you can put the text directly in the HTML instead of in a long JS string. Again just an fyi, this way is perfectly fine if you prefer it.

Reply:
I just was trying to do it in JS to play around with it. I might try that other way though with something else.

Reply:
Ya of course. Just wanted to let you know as an option.